content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Monitoring Page Performance
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "/real_user_monitoring/explorer/"
+    tag: "Documentation"
+    text: "Explore your views within Datadog"
+  - link: "/real_user_monitoring/explorer/analytics/"
+    tag: "Documentation"
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog as the page finishes loading, but might be empty for a few seconds before they appear in the Datadog UI.
+
+## Identifying performance bottlenecks
+
+To find what's causing a page to under-perform:
+
+1. Set up and configure [RUM Browser monitoring][1] for your application.
+2. Open the [Performance Overview dashboard][2]. It provides you with a high level view of your monitored pages, and you can apply filters to find pages with performance problems.
+3. When you find a page with problem, click the event to open it in the [RUM Explorer][3].
+4. Open the event side panel to see what is causing the performance issue with the help of the waterfall.
```

Comment:
Clicking on one view in the RUM explorer lets you access performance information and a lot of details about that specific page view. Troubleshoot the performance bottleneck/issue with the waterfall.